### PR TITLE
[Doc] Update 22.08 documentation

### PR DESCRIPTION
**Pull request by viadea**
Update the 22.08 documentation.
1. Add release notes for 22.08
2. Change the diagram from Spark 3.0 to Spark 3.x

Note: In this PR, I did not add the download links for 22.08 jars or modify the Tool jar links to avoid link check failure issues.
I will do it right before the final merge PR in another PR.

### CODE REVIEW

File: docs/download.md [hunk 1]
```diff
@@ -18,6 +18,47 @@ cuDF jar, that is either preinstalled in the Spark classpath on all nodes or sub
...
+*For Cloudera and EMR support, please refer to the
+[Distributions](./FAQ.md#which-distributions-are-supported) section of the FAQ.
+
+### Release Notes
+New functionality and performance improvements for this release include:
+* Rocky Linux 8 support
+* Ability to build Spark RAPIDS jar using JDK 11 
+* ZSTD parquet read support
+* Binary read support from parquet
+* Apache iceberg 0.14 support
```
Comment (jlowe) [inline docs/download.md:50]:
```suggestion
* Apache Iceberg 0.13 support  
```

**Review by jlowe — APPROVED** [review]

File: docs/download.md [hunk 2]
```diff
@@ -18,6 +18,47 @@ cuDF jar, that is either preinstalled in the Spark classpath on all nodes or sub
...
+for your hardware's minimum driver version.
+
+*For Cloudera and EMR support, please refer to the
+[Distributions](./FAQ.md#which-distributions-are-supported) section of the FAQ.
+
+### Release Notes
+New functionality and performance improvements for this release include:
+* Rocky Linux 8 support
+* Ability to build Spark RAPIDS jar using JDK 11 
+* ZSTD parquet read support
```
Comment (sameerz) [inline docs/download.md:48]:
```suggestion
* Zstandard Parquet and ORC read support
```

File: docs/download.md [hunk 3]
```diff
@@ -18,6 +18,47 @@ cuDF jar, that is either preinstalled in the Spark classpath on all nodes or sub
...
+*Some hardware may have a minimum driver version greater than v450.80.02+.  Check the GPU spec sheet
+for your hardware's minimum driver version.
+
+*For Cloudera and EMR support, please refer to the
+[Distributions](./FAQ.md#which-distributions-are-supported) section of the FAQ.
+
+### Release Notes
+New functionality and performance improvements for this release include:
+* Rocky Linux 8 support
+* Ability to build Spark RAPIDS jar using JDK 11
```
Comment (sameerz) [inline docs/download.md:47]:
```suggestion
* Ability to build Spark RAPIDS jars for Java versions 9+
```

File: docs/download.md [hunk 4]
```diff
@@ -18,6 +18,47 @@ cuDF jar, that is either preinstalled in the Spark classpath on all nodes or sub
...
+
+	GPU Models: NVIDIA V100, T4 and A2/A10/A30/A100 GPUs
+
+Software Requirements:
+
+	OS: Ubuntu 18.04, Ubuntu 20.04 or CentOS 7, Rocky Linux 8
+
+	CUDA & NVIDIA Drivers*: 11.x & v450.80.02+
+
+	Apache Spark 3.1.1, 3.1.2, 3.1.3, 3.2.0, 3.2.1, 3.3.0, Databricks 9.1 ML LTS or 10.4 ML LTS Runtime and GCP Dataproc 2.0
```
Comment (sameerz) [inline docs/download.md:34]:
```suggestion
	Apache Spark 3.1.1, 3.1.2, 3.1.3, 3.2.0, 3.2.1, 3.2.2, 3.3.0, Databricks 9.1 ML LTS or 10.4 ML LTS Runtime and GCP Dataproc 2.0
```

**Review by sameerz — COMMENTED** [review]
Other changes:
* Update the FAQ to note we support Spark 3.2.2
* There is a table formatting issue on gh-pages in https://nvidia.github.io/spark-rapids/docs/compatibility.html#csv-ansi-day-time-interval, possibly due to a lack of a blank line above the text "Currently, the RAPIDS Accelerator only supports the ANSI style." in https://raw.githubusercontent.com/NVIDIA/spark-rapids/branch-22.08/docs/compatibility.md . 
* In https://github.com/NVIDIA/spark-rapids/blob/branch-22.08/docs/spark-profiling-tool.md , change `Profiling tool` to `Profiling Tool` to be consistent with other page titles at https://nvidia.github.io/spark-rapids/

**Comment by sameerz** [comment]
Can we update https://github.com/NVIDIA/spark-rapids/blob/branch-22.08/docs/configs.md (which is a generated file) to remove `--conf spark.rapids.sql.incompatibleOps.enabled=true` since it is now true by default?

**Comment by nvliyuan** [comment]
if a customer setup a databricks cluster, then log in to one cluster node and run some codes through pyspark/spark-shell scripts with rapids-plugin,  it will complain that the spark shim loader can not be found, because it is up to `spark.databricks.clusterUsageTags.sparkVersion` to decide which db shim should be load, so customers need to set an additional config `--conf spark.rapids.shims-provider-override=com.nvidia.spark.rapids.shims.spark3XXdb.SparkShimServiceProvider`; this is set automatically on DB notebook but not scripts

File: docs/download.md [hunk 5]
```diff
@@ -18,6 +18,47 @@ cuDF jar, that is either preinstalled in the Spark classpath on all nodes or sub
...
+
+### Release Notes
+New functionality and performance improvements for this release include:
+* Rocky Linux 8 support
+* Ability to build Spark RAPIDS jar using JDK 11 
+* ZSTD parquet read support
+* Binary read support from parquet
+* Apache Iceberg 0.13 support  
+* Array function support: array_intersect, array_union, array_except and arrays_overlap
+* Function from_json support
```
[the remark this answers is not in the document]
Comment (viadea) [reply docs/download.md]: Removed.

[the remark this answers is not in the document]
Comment (viadea) [reply docs/download.md]: Fixed all above.

**Comment by viadea** [comment]
> 

Good information. But I feel Databricks users may not really want to SSH to the nodes and run spark-shell.

**Comment by viadea** [comment]
> Can we update https://github.com/NVIDIA/spark-rapids/blob/branch-22.08/docs/configs.md (which is a generated file) to remove `--conf spark.rapids.sql.incompatibleOps.enabled=true` since it is now true by default?

@sameerz  I modified below 2 files to use another parameter `spark.rapids.sql.concurrentGpuTasks` to replace `spark.rapids.sql.incompatibleOps.enabled`:
./spark2-sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala
./sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

Also fixed one typo : spark -> spark-shell

BTW: I am not 100% sure why there is another ./spark2-sql-plugin but i changed both. If there is any concern, please let me know.

**Review by sameerz — APPROVED** [review]
Looks good.  I am going to remove the `[skip ci]` since there are changes to RapidsConf.scala.

**Comment by sameerz** [comment]
build

**Pull request merged**